src/Integration.UnitTests/ProjectToLanguageMapperTests.cs

```diff
@@ -90,36 +87,27 @@ public void GetAllBindingLanguagesForProject_SupportedProjectType_CPP_ReturnsCor
         }
 
         [TestMethod]
-        [DataRow("c:\\some directory\\CMakeLists.txt")]
-        [DataRow("c:\\some directory\\sub\\CMakeLists.txt")]
-        [DataRow("c:\\some directory\\sub\\folder\\CMakeLists.txt")]
-        public void GetAllBindingLanguagesForProject_OpenAsFolderProject_HasCmakeFiles_CFamilyLanguages(string cmakeListsLocation)
+        public void GetAllBindingLanguagesForProject_CMakeProject_CFamilyLanguages()
         {
-            var folderWorkspaceService = SetupOpenAsFolder("c:\\some directory");
-            var fileSystem = new MockFileSystem();
-            fileSystem.AddDirectory("c:\\some directory");
-            fileSystem.AddFile(cmakeListsLocation, new MockFileData(""));
+            var folderWorkspaceService = SetupCMakeProject(true);
```

Comment:
The variable name 'folderWorkspaceService' needs to be updated in all of the tests in this class.